src/main/java/seedu/canoe/logic/commands/AddStudentToTrainingCommand.java

```diff
@@ -84,11 +84,14 @@ public CommandResult execute(Model model) throws CommandException {
 
         List<Training> lastShownList = model.getFilteredTrainingList();
         List<Student> studentList = model.getFilteredStudentList();
+
+        //Checks if the user specified any students to add
         if (studentsToAdd == null) {
             LOGGER.warning("User input invalid");
             throw new CommandException(MESSAGE_NO_STUDENTS_SPECIFIED);
         }
```

Comment:
This is redundant null check because the constructor already has `requireNonNull(studentsToAdd)`, instead it should check for `studentsToAdd.isEmpty()`